### PR TITLE
Feature/add oep models ppr

### LINKED ISSUE #42: Missing table definition in oep_models
Add the supply.ego_dp_res_powerplant table from OEDB as SQLAlchemy table definition to oep_models.

### PR DESCRIPTION
**Pull request by jh-RLI**
Closes #42 

This creates the table definition for the [supply.ego_dp_res_powerplant](https://openenergy-platform.org/dataedit/view/supply/ego_dp_res_powerplant) table from OEDB using SQLAlchemy reflection.

This is needed to query the table.

### CODE REVIEW

**Comment by jh-RLI** [comment]
@Bachibouzouk can you help me fixthe Travis CI error?

Edit: got it

File: oep_models.py [hunk 1]
```diff
@@ -26,10 +27,11 @@
 
 
 # ##########################################SQLAlchemy setup########################################
-SCHEMA = 'model_draft'
+SCHEMA_1 = 'model_draft'
+SCHEMA_2 = 'supply'
 engine = sah.get_engine('oep_engine')
-metadata = MetaData(schema=SCHEMA, bind=engine, reflect=True)
-
+metadata_1 = MetaData(schema=SCHEMA_1, bind=engine)
```
Comment (jh-RLI) [inline oep_models.py:33]: removed reflect=True since it slows the wam down reflect=False is default

File: oep_models.py [hunk 2]
```diff
@@ -162,8 +162,4 @@ class ResPowerPlantRegister(Base):
```
Comment (Bachibouzouk) [inline oep_models.py:162]: you can install flake8 and pylint locally to run the tests locally :)

(`pip install pylint flake8`)

**Review by Bachibouzouk — APPROVED** [review]
Good job @jh-RLI  :)

You want to add the query in a subsequent PR? I'll approve the PR and you can decide if you want to add more commits with the query and the template or merge now :)

**Comment by jh-RLI** [comment]
Yes the query will be in another pr.

**Pull request merged**